clarifai/datasets/upload/examples/image_classification/food-101/dataset.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from clarifai.datasets.upload.base import ClarifaiDataLoader
+from clarifai.datasets.upload.features import VisualClassificationFeatures
+
+
+class Food101DataLoader(ClarifaiDataLoader):
```

Comment:
Maybe we should cut the food-101 images to 5 per class(folder) since it's just examples. Users can download the full dataset locally if they wish to upload the whole of it.

Reply:
Also you should fix indentation for some of the docstrings.